### ISSUE TITLE
Subcollection

### ISSUE THREAD

**Issue by fokkerone** [linkwarden/linkwarden#818]
**Is your feature request related to a problem? Please describe.**
public/shared Subcollections does not have the same access right as parent collection, frankly there is no difference with a complete new collection,. 

**Describe the solution you'd like**
Subcollections should have same ACL/Users as parent collection

**Describe alternatives you've considered**
-
**Additional context**
-

**Comment by Danie10** [comment]
Agreed, it is a bit difficult to share out 10 separate links, as I have tried to group sub-collections under my Public shares to give out one link, that people can see there are a few sub-collections also to navigate into.

**Comment by landure** [comment]
I completely agree. For example, a public collection should display its sub-collections. As it is, sharing a well organized collection with many sub-collections is a hassle. This is the missing feature preventing linkwarden to be team friendly.

**Comment by Danie10** [comment]
Hoping this will still be looked at - a sub-collection by its definition is a something linked to the parent. But the way it is currently shared publicly, they exist as completely separate collections only. Only I see the sub-collection links when I view the parent collection. Anyone I share my parent collection with publicly, never sees the sub-collections, even though they were shared too.
